ControlMeasurements/Startup.cs

```diff
@@ -53,7 +53,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseExceptionHandler("/Home/Error");
                 app.UseHsts();
             }
-
+            app.UseDefaultFiles();
```

Comment:
Po co to dodałeś? Gdzieś jest to wymagane?

Reply:
Chciałem cos sprawdzić - miałem usunać.